src/main.py

```diff
@@ -48,7 +48,11 @@
                     'type': 'string'
                 }
             }
-        }
+        },
+        'log_level': {
+            "type": "string",
+            "enum": ["CRITICAL", "ERROR", "WARNING", "INFO", "DEBUG", "NOTSET"],
```

Comment:
Remove the trailing comma (because `line too long (80 > 79 characters)flake8(E501)`)

Reply:
It looks it is preferred to hierarchize the enum's array because the existing code does that (e.g. https://github.com/cohalz/CertUpdater/blob/c73b58f682b5fba699467c467b893670b83ff373/src/main.py#L15)
So it followed that @ https://github.com/cohalz/CertUpdater/pull/6/commits/168e4eb63be70fe998a154b3ee885b7e876d2096